parsers/BR.py

```diff
@@ -55,62 +63,51 @@ def get_data(session: Optional[Session]):
     return json_data
 
 
-def production_processor(json_data, zone_key: str) -> tuple:
+def production_processor(
+    json_data: dict, zone_key: str
+) -> Tuple[datetime, ProductionMix]:
     """Extracts data timestamp and sums regional data into totals by key."""
 
-    dt = arrow.get(json_data["Data"])
+    dt = arrow.get(json_data["Data"]).datetime
     totals = defaultdict(lambda: 0.0)
 
     region = REGIONS[zone_key]
     breakdown = json_data[region]["geracao"]
     for generation, val in breakdown.items():
         totals[generation] += val
 
-    # BR_CS contains the Itaipu Dam.
-    # We merge the hydro keys into one, then remove unnecessary keys.
-    totals["hydro"] = (
-        totals.get("hidraulica", 0.0)
-        + totals.get("itaipu50HzBrasil", 0.0)
-        + totals.get("itaipu60Hz", 0.0)
-    )
-    entries_to_remove = {"hidraulica", "itaipu50HzBrasil", "itaipu60Hz", "total"}
-    mapped_totals = {
-        GENERATION_MAPPING.get(name, "unknown"): val
-        for name, val in totals.items()
-        if name not in entries_to_remove
-    }
+    del totals["total"]
```

Comment:
If we are getting a total value should we not also add a TotalProductionList (total generation parser)?

I can do that in a seperate PR if you want to get this merged first.

Reply:
What about consumption? I think "cargaVerificada" is load and if we subtract the exports from that we should get the consumption data.